assimp.diff

```diff
@@ -0,0 +1,16 @@
+diff a/third_party/assimp/code/CMakeLists.txt b/third_party/assimp/code/CMakeLists.txt
+--- a/third_party/assimp/code/CMakeLists.txt
++++ b/third_party/assimp/code/CMakeLists.txt
+@@ -1176,13 +1176,6 @@ ADD_LIBRARY(assimp::assimp ALIAS assimp)
+ 
+ TARGET_USE_COMMON_OUTPUT_DIRECTORY(assimp)
+ 
+-# enable warnings as errors ########################################
+-IF (MSVC)
+-  TARGET_COMPILE_OPTIONS(assimp PRIVATE /WX)
+-ELSE()
+-  TARGET_COMPILE_OPTIONS(assimp PRIVATE -Werror)
```

Comment:
What was the problem here? Should we file a bug on Assimp?

Reply:
https://github.com/pca006132/manifold/runs/6302903855?check_suite_focus=true#step:6:1258

Not sure why this happens in CI, I don't get this error when compiling it locally (but can reproduce when running it in the docker instance using the exact same image). I guess some code are probably conditionally compiled and are omitted in the CI environment and cause unused variables.

Reply:
I got this error locally until I applied the patch, FWIW.